Tasks/WindowsMachineFileCopy/Utility.ps1

```diff
@@ -61,7 +61,7 @@ function Validate-Null(
     $value = $value.Trim()
     if(-not $value)
     {
-        ThrowError -errorMessage (Get-LocalizedString -Key "Parameter '{0}' cannot be null or empty." -ArgumentList $variableName)
+        ThrowError -errorMessage (Get-VstsLocString -Key "WFC_ParameterCannotBeNullorEmpty" -ArgumentList $variableName)
     }
 }
```

Comment:
IMO this file is not required at all as now we should not deal with any environment objects here

Reply:
Removed unnecessary methods